platform/nativescript/runtime/components/list-view.js

```diff
@@ -13,6 +20,9 @@ export default {
         templateSelector: {
             type: Function,
             default: () => 'default'
+        },
+        separatorColor: {
+            type: String
```

Comment:
The native implementation already has a default (grey-ish) - no need to duplicate. And IMO we only need `required` in case it's `true` (which it's not).